lms/djangoapps/bulk_email/tasks.py

```diff
@@ -418,12 +419,32 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
         'failed' count above.
     """
     # Get information from current task's request:
+    parent_task_id = InstructorTask.objects.get(pk=entry_id).task_id
     task_id = subtask_status.task_id
+    total_recipients = len(to_list)
+    recipient_no = 0
+    total_recipients_successful = 0
+    total_recipients_failed = 0
+    recipients_info = Counter()
+
+    log.info("=============== BulkEmail ==> start logging for send email task ===============")
```

Comment:
I don't think this line will be helpful.  There will be up to four send_email tasks running in parallel, and logging in parallel.  This won't be very readable in the log file.  The line below should be sufficient.